.github/workflows/livecodes-preview.yml

```diff
@@ -0,0 +1,14 @@
+name: livecodes
+
+on: [pull_request]
+
+jobs:
+  build_and_prepare:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: live-codes/preview-in-livecodes@v1
+        with:
+          install-command: "yarn install --frozen-lockfile --check-files"
+          build-command: "yarn build"
+          base-url: "https://{{LC::REF}}.preview-in-livecodes-demo.pages.dev"
```

Comment:
you may want to change this to:

```yml
base-url: "https://jotai-git-{{LC::REF}}-pmndrs.vercel.app"
```

in case you want to use `{{LC:TO_URL}}`, instead of data urls.

please make sure that CORS is enabled on your deployment to allow loading assets in the playground.

Reply:
But we don't deploy anything for livecodes to vercel.app. I think DataURL is safer.